demo-shell/src/app/components/card-view/card-view.component.html

```diff
@@ -6,7 +6,8 @@ <h1>CardView Component</h1>
             [properties]="properties"
             [editable]="isEditable"
             [displayClearAction]="showClearDateAction"
-            [displayNoneOption]="showNoneOption">
+            [displayNoneOption]="showNoneOption"
+            [displayLabelForChips]="showLabelForMultiValuedChip">
```

Comment:
I would change `showLabelForMultiValuedChip` to `showLabelForChips` too, to avoid additional mental steps

Reply:
Renamed